src/BinSkim.Rules/PERules/BA2004.EnableSecureSourceCodeHashing.cs

```diff
@@ -104,8 +107,11 @@ public void AnalyzeNativeBinaryAndPdb(BinaryAnalyzerContext context)
                 return;
             }
 
-            var compilandsBinaryWithOneOrMoreInsecureFileHashes = new List<ObjectModuleDetails>();
             var compilandsLibraryWithOneOrMoreInsecureFileHashes = new List<ObjectModuleDetails>();
+            var compilandsLibraryWithOneOrMoreNoFileHashes = new List<ObjectModuleDetails>();
```

Comment:
>compilandsLibraryWithOneOrMoreNoFileHashes [](http://example.com/codeflow?start=16&length=42)

wouldnt' a name like 'compilandsLibraryWithOneOrMoreMissingFileHashes' be a little clear?

Rather than a plurality of NoFileHashes?